src/main/java/com/foundation/salesforce/core/utils/EndPoints.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * @(#) EndPoints.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.core.utils;
+
+/**
+ * EndPoints class
+ *
+ * @author Cristian Lujan
+ * @version 1.0
+ */
+public class EndPoints {
+
+    /**
+     * Constructor protected.
+     */
+    private EndPoints(){
+
+    }
+    /**
+     * Endpoint of account.
+     */
+    public final static String ACCOUNT_ENDPOINT = "/v46.0/sobjects/Account";
+
+    /**
+     * Endpoint of Opportunity.
+     */
+    public final static String OPPORTUNITY_ENDPOINT = "/v46.0/sobjects/Opportunity";
+
+    /**
+     * Endpoint of Task.
+     */
+    public final static String TASK_ENDPOINT = "/services/data/v46.0/sobjects/Task";
+}
```

Comment:
The class needs a space after the bracket please

Reply:
Fixed.